lib/paths-cache.js

```diff
@@ -214,20 +212,24 @@ export default class PathsCache extends EventEmitter {
     this._cancelled = false
     this.emit("rebuild-cache")
 
-    return exec("find", ["--version"], { maxBuffer: MAX_STRING_LENGTH }).then(
-      // `find` is available
-      () => this._buildInitialCacheWithFind(),
-      // `find` is not available
-      () => this._buildInitialCache()
-    )
+    let cacheFiles
+    this._buildInitialCacheWithGlob()
+      .then((files) => {
+        cacheFiles = files
+      })
+      .catch((e) => {
+        console.error(e)
+        cacheFiles = this._buildInitialCacheWithAtom()
+      })
+    return cacheFiles
```

Comment:
`cacheFiles` will always be `undefined` since you are not awaiting `this._buildInitialCacheWithGlob()`

```suggestion
    return this._buildInitialCacheWithGlob()
      .catch((e) => {
        console.error(e)
        return this._buildInitialCacheWithAtom()
      })
```

Reply:
Fixed!